Gopkg.toml

```diff
@@ -0,0 +1,50 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/microendpoint"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/microerror"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/microkit"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/micrologger"
+
+[[constraint]]
+  name = "github.com/spf13/viper"
+  version = "=1.2.1"
```

Comment:
FMI did `dep` decide on this or you? AFAIK this should just come with `microkit`.

Reply:
I added this because otherwise it defaults to `^`. I didn't realize it actually comes from dependency as well so I remove it.

Reply:
Lets pin it properly in `microkit` when this is important.